src/main/java/de/spinscale/elasticsearch/ingest/opennlp/EntityFinderService.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright [2016] [Alexander Reelsen]
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package de.spinscale.elasticsearch.ingest.opennlp;
+
+import java.util.List;
+
+public interface EntityFinderService {
```

Comment:
Ah, i forgot. In Java you can mock classes, e.g. with Mockito.

Ok, will do that...